libs/schemas/src/entities/comment.schemas.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
+import { PG_INT } from '../utils';
+
+export const CommentVersionHistory = z.object({
+  id: PG_INT.optional(),
+  comment_id: PG_INT,
+  text: z.string(),
+  timestamp: z.date().optional(),
```

Comment:
model indicates that timestamp is required?

Reply:
I had named them timestamp since they are not like our usual created_at/updated_at entities, since these entities represent the immutable history.